modules/gitops-additional-steps-for-disconnected-clusters.adoc

```diff
@@ -0,0 +1,43 @@
+[id="additional-steps-disconnected-clusters_{context}"]
+= Additional steps for disconnected clusters
+
```

Comment:
Minor verb change: `Follow the additional steps to integrate Keycloak with OpenShift Oauth Login.` 

It cuts down on word count and is more to the point. NBD if you want to keep the original phrasing since that works well too!